Source/Krypton Components/Krypton.Navigator/EventArgs/PageDropEventArgs.cs

```diff
@@ -37,7 +37,7 @@ public class PageDropEventArgs : CancelEventArgs
         public KryptonPage? Page
         {
             get => _page;
-            set => _page = Page;    // TODO: What is this supposed to be doing ?
+            set => _page = value;    // TODO: What is this supposed to be doing ?
```

Comment:
Comment can be removed now ;-)

Reply:
Done, I wondered why it was doing this while sifting through